src/chains-config/metadata-consts/kusamaConsts.ts

```diff
@@ -0,0 +1,28 @@
+import { MetadataConsts } from '../../types/chains-config';
+import { extrinsicBaseWeight, perClass } from './substrateConsts';
+
+export const kusamaDefinitions: MetadataConsts[] = [
+	{
+		runtimeVersions: [
+			1062,
+			2005,
+			2007,
+			2008,
+			2011,
+			2012,
+			2013,
+			2015,
+			2019,
+			2022,
+			2023,
+			2024,
+			2025,
+			2026,
+		],
+		extrinsicBaseWeight,
+	},
+	{
+		runtimeVersions: [2027, 2028, 2029],
```

Comment:
What happens for future versions here? `2030` comes along and does not appear here; shouldn't we assume it's `perClass` too until we know it's not? Or maybe that's taken care of elsewhere?

Reply:
Thats a good point. Currently the idea is before every runtime upgrade we just add the new runtime in with the weekly releases. Fortunately with the way it works, and is designed, is even if its not there the service will notice that and it will decorate the meta data and then cache the value inside of the `blockWeightStore`.

Reply:
> Fortunately with the way it works, and is designed, is even if its not there the service will notice that and it will decorate the meta data and then cache the value inside of the `blockWeightStore`.

Oh I missed that, can you point me to where in the code this happens?

Reply:
Yea ofcourse the exact line is [here](https://github.com/paritytech/substrate-api-sidecar/blob/fix-metadata-regression/src/services/blocks/BlocksService.ts#L523). The `||=` operation says if the statement on the left hand side does not exist set it equal to the right hand operation, else dont do anything as it already exists.